pkg/pipelines/init.go

```diff
@@ -115,6 +115,7 @@ func Init(o *InitOptions, fs afero.Fs) error {
 	}
 	if o.GitOpsWebhookSecret == "" {
 		gitSecret, err := secrets.GenerateString(webhookSecretLength)
+		fmt.Println("Was in this webhook block")
```

Comment:
The `print` used for debugging can be removed

Reply:
done